scripts/ansible/roles/windows/az-dcap-client/vars/windows.yml

```diff
@@ -15,3 +15,22 @@ lc_driver:
   reg_path: HKLM:\SYSTEM\CurrentControlSet\Services\sgx_lc_msr\Parameters
   reg_key: SGX_Launch_Config_Optin
   reg_value: 1
+
+validation_directories:
+  - "C:\\openenclave\\prereqs\\nuget\\curl"
+  - "C:\\openenclave\\prereqs\\nuget\\DCAP_Components"
+  - "C:\\openenclave\\prereqs\\nuget\\EnclaveCommonAPI"
+  - "C:\\openenclave\\prereqs\\nuget\\Microsoft.Azure.DCAP.Client"
+  - "C:\\openenclave\\prereqs\\nuget\\openssl"
+  - "C:\\openenclave\\prereqs\\nuget\\zlib"
+
```

Comment:
Why do we need openssl? For Windows, we have been using openssl from the Git Bash directory.
Why is zlib needed?

Reply:
None of these are relevant. The bad rebase has been dropped.

Reply:
Great. I later saw that this was already in the master.